pkg/safe/routine.go

```diff
@@ -20,7 +21,7 @@ type routineCtx func(ctx context.Context)
 // Pool is a pool of go routines
 type Pool struct {
 	routines    []routine
```

Comment:
You should do the same with this slice?

Reply:
Yes we should. But maybe in next PR, since it is not needed to fix #6125 ?

Reply:
Done